openmetadata-service/src/main/java/org/openmetadata/service/apps/ConfigurationReader.java

```diff
@@ -0,0 +1,74 @@
+package org.openmetadata.service.apps;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+import joptsimple.internal.Strings;
```

Comment:
@sushi30 not sure if it works , but we can do something like below a try

```
     ObjectMapper objectMapper = Jackson.newObjectMapper();
    Validator validator = Validators.newValidator();
    YamlConfigurationFactory<AppPrivateConfig> factory =
        new YamlConfigurationFactory<>(
                AppPrivateConfig.class, validator, objectMapper, "dw");
    config =
        factory.build(
            new SubstitutingSourceProvider(
                new FileConfigurationSourceProvider(), new EnvironmentVariableSubstitutor(false)),
            configFilePath);
```
We are using this in Dropwizard

Reply:
addressed